src/crepe/api/Text.h

```diff
@@ -17,22 +17,8 @@ class Text : public UIObject {
 public:
 	//! Text data that does not have to be set in the constructor
 	struct Data {
-		/**
-		 *  \brief fontsize for text rendering
-		 * 
-		 * \note this is not the actual font size that is loaded in.
-		 * 
-		 * Since SDL_TTF requires the font size when loading in the font it is not possible to switch the font size.
-		 * The default font size that is loaded is set in the Config.
-		 * Instead this value is used to upscale the font texture which can cause blurring or distorted text when upscaling or downscaling too much.
-		 */
-		unsigned int font_size = 16;
-
-		//! Layer sorting level of the text
-		const int sorting_in_layer = 0;
-
-		//! Order within the sorting text
-		const int order_in_layer = 0;
+		//! variable indicating if transform is relative to camera(false) or world(true)
+		bool world_space = false;
```

Comment:
kun je de offset optioneel maken door hem in de struct te zetten?

Reply:
@JaroWMR Dit gaat niet omdat `offset` in `UIObject` op is geslagen. Wat wel zou kunnen, is `offset` achteraan de constructor plaatsen met default waarde (waardoor `offset` optioneel wordt om in te voeren).